applications/vanilla/models/class.categorymodel.php

```diff
@@ -2801,4 +2805,126 @@ public function recalculateTree() {
     public static function flattenTree($categories) {
         return self::instance()->collection->flattenTree($categories);
     }
+
+    /**
+     * Adjust the aggregate post counts for a category.
+     *
+     * @param int|string|object|array $category
+     * @param int $offset
```

Comment:
Documenting what this variable does would answer the obvious question: "Adjust how?"